### ISSUE TITLE
Character counter needs z-index to be visible at file view

### ISSUE THREAD

**Issue by jakobploens** [getkirby/panel#900]

**Comment by jakobploens** [comment]
In v2.3.2, the counter isn't applied at all, for both `max` and `maxLength`.

**Comment by lukasbestle** [comment]
Which browser on which OS are you using? Could you please post the full blueprint?

**Comment by jakobploens** [comment]
FirefoxDeveloperEdition on OS X El Capitan. Just tested it in Safari, it works there without problems. Chrome also needs the custom stylesheet. All browsers are the latest available version.

My blueprint:

```
# site/blueprints/home.yml

title: Startseite
icon:  home
pages: false
files:
    sortable: true
    fields:
        link:
            label: Link
            type:  url
        alt: alt
        text:
            label: Text
            type:  textarea

# site/blueprints/fields/alt.yml

label: Alternativ-Text
type:  text
validate:
    maxLength: 50

```

Direct implementation without the field definition doesn't change anything. The markup seems to be correct, so I guess it's really just a CSS problem…

**Comment by lukasbestle** [comment]
Thanks, I can reproduce this now, even in Safari.

The `z-index` of the counter is [set to `-1` explicitly](https://github.com/getkirby/panel/blob/master/app/fields/text/assets/css/counter.css#L3).
@iksi What was the reason for that? I don't want to remove it if it's still required for something.

**Comment by iksi** [comment]
I think that is some legacy code to make sure it never overlays a form field’s `<label>` and can be safely removed.

**Comment by lukasbestle** [comment]
Thank you very much for the reply. We will fix this issue in the next release.

**Comment by lukasbestle** [comment]
The fix will be included in Kirby 2.4.
